.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Proposed changes
+
+Include description and references to tickets or issues that this PR is related to
+
+## Checklist prior to adding reviewers
+
+_Put an `x` in the boxes that apply._
+
+- [ ] I have run this successfully locally using `./build.ps1 compose`
+- [ ] I have added tests for my code and/or updated existing tests where my changes impacts them, or there is a valid explanation for not including tests in this PR
+- [ ] I have added necessary documentation (if appropriate)
+- [ ] I created this branch from main and have integrated the latest changes
```

Comment:
Branches should be created from `development`, not `master` / `main`.

Reply:
This is an intention to migrate away from the current setup which supports a single set of environments to enable support for more environments and make them more composable that the current automation allows in non prod.

The usage of a persistent development branch adds complexity to the release process and encourages batching of changes to be deployed rather than encouraging small incremental change.

With the proposed change, there would be a `main`/`master` branch which represents the current latest state of the project. Feature branches are made from that directly and tested in isolation from one another using supporting automation. The versioning approach using sem ver + the ability to publish a version of the repo as a PR version means that when deploying, we can reference a specific version of code to compose the environment in a very granular fashion without resorting to merging code and potentially reverting it if it isn't desirable/doesn't work.